docs/user_guide/celeborn_ratis_shell.md

```diff
@@ -0,0 +1,154 @@
+---
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+
+# Celeborn Ratis-shell
+
+[Ratis-shell](https://github.com/apache/ratis/blob/master/ratis-docs/src/site/markdown/cli.md) is the command line interface of Ratis.
+Celeborn uses Ratis to implement the HA function of the master, Celeborn directly introduces ratis-shell package into the project
+then it's convenient for Celeborn Admin to operate the master ratis service.
+
+> **Note**:
+> Ratis-shell is currently only **experimental**.
+> The compatibility story is not considered for the time being.
+
+## Availability
+|  Version  | Available in src tarball? | Available in bin tarball? |
+|:---------:| :-----------------------: |:-------------------------:|
+|  < 0.3.0  | No                        |            No             |
+| \>= 0.3.0 | Yes                       |            Yes            |
+
+## Setting up the Celeborn ratis-shell
+
+Celeborn directly introduce the ratis-shell into the project, users don't need to set up ratis-shell env from ratis repo.
+Usr can directly download the Celeborn source tarball from [Download](https://celeborn.apache.org/download/#security) and
+build the Celeborn accoriding to [build_and_test](https://celeborn.apache.org/community/contributor_guide/build_and_test/)
+or just down load the bin tarball from [Download](https://celeborn.apache.org/download/#security) to get the binary package `apache-celeborn-<VERSION>-bin.tgz`.
+
+After get the binary package `apache-celeborn-<VERSION>-bin.tgz`:
+```
+$ tar -C <DST_DIR> -zxvf apache-celeborn-<VERSION>-bin.tgz
+$ mv <DST_DIR>/apache-celeborn-<VERSION>-bin <DST_DIR>/celeborn
+```
+
+Export the `CELEBORN_HOME`, `CELEBORN_CONF_DIR` and `CELEBORN_LOG_DIR` environment variable and add the bin directory to the `$PATH`.
+```
+$ export CELEBORN_HOME=<DST_DIR>/celeborn
+$ export CELEBORN_CONF_DIR=$CELEBORN_HOME/conf
+$ export CELEBORN_LOG_DIR=$CELEBORN_HOME/logs
```

Comment:
This three are encouraged to be put into `celeborn-env.sh` instead of declaring globally